pkg/utils/urls/urls.go

```diff
@@ -25,6 +25,10 @@ const (
 	Client        // Client == 1
 )
 
+func GenerateFileShareURL(urlType int, tenantId string, in ...string) string {
+        return generateURL("file/shares", urlType, tenantId, in...)
```

Comment:
Same as above comment, it will be good to keep URLs as constant in some file and refer that file.